### DISCUSSION TITLE
Usage of HTTPS in Azure Function with plumber

### DISCUSSION THREAD

**Discussion by rhuettl** [revodavid/R-custom-handler#5] [General]
Hello team,

One ore question: Is there an option to configure HTTPS usage for the API communication? We like to encrypt the communication via HTTPS if possible.

Thanks, best regards,
Ralf

**Comment by revodavid** [comment]
I strongly suspect so, but I don't know the ins-and-out of handling the certificates. @anthonychu might know.

**Reply by rhuettl** [reply]
Thanks. Seems that I can use the default SSL/TLS features provided by functions. This way the communication is secured to the function, internally it hasn't so be secure (Port 80 forward) :-)
